runtime-modules/forum/src/lib.rs

```diff
@@ -183,17 +183,17 @@ pub trait Config:
 }
 
 /// Upper bounds for storage maps and double maps. Needed to prevent potential block exhaustion during deletion, etc.
-/// MaxSubcategories, and MaxCategories should be reasonably small because when the category is deleted
+/// MaxDirectSubcategoriesInCategory, and MaxTotalCategories should be reasonably small because when the category is deleted
 /// all of it's subcategories with their threads and posts will be iterated over and deleted.
```

Comment:
Almost the entire comment is wrong, there is no iteration over subcategories/threads during category removal. Removing a category requires the `category.num_direct_threads` and `category.num_direct_subcategories` to be `0`.
```suggestion
/// Upper bounds for storage maps and double maps.
```